document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImpl.java

```diff
@@ -32,17 +36,27 @@ public DocumentInfoResponse getAccountsData(String resourceLink) throws HandlerE
             throw new HandlerException(e.getMessage(), e.getCause());
         }
 
-        // TODO: The accountsType variable shall be used in the implementation of abridged logic
         AccountType accountsType = getAccountType(accounts);
 
+        String abridgedAccountLink = getAccountLink(accounts, accountsType);
+        try {
+            // TODO: abridgedAccountData will be used as part of the implementation of converting
+            // TODO: the data object to a json string
+            AbridgedAccountsApi abridgedAccountData = accountsService.getAbridgedAccounts(abridgedAccountLink);
+        } catch (ServiceException e) {
+            throw new HandlerException(e.getMessage(), e.getCause());
```

Comment:
Can you log a message here with the available details of the account that was affected by this error (account ID, account type, etc) please?

Reply:
Refined logging will be done in a separate PR as explain in the PR description as well as the refined exception handling. I have submitted basic logging so this should be fine, but remember that the logging isn't going to be complete in these PRs until there is a PR that addressing logging entirely.

Reply:
Logging is a non-functional requirement that should exist in each PR. I agree that we should review/improve it over time as we use the logs and it won't be perfect the first time but it needs to be included.